### PR TITLE
Start default network after firewalld service restart

### PR DESCRIPTION
**Pull request by yalzhang**
Add 2 test scenario about start default network after firewalld restart.

Signed-off-by: Yalan <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
This is to automate RHEL-246778.
run locally pass:
$ avocado run --vt-type libvirt virsh.net_start.normal_test.restart_firewalld.valid_netname  virsh.net_start.normal_test.stop_start_firewalld.valid_netname
JOB ID     : 2bcc23fa53c9aa6f4427a82ecbc99dfb877e515c
JOB LOG    : /root/avocado/job-results/job-2021-07-12T17.41-2bcc23f/job.log
 (1/2) type_specific.io-github-autotest-libvirt.virsh.net_start.normal_test.restart_firewalld.valid_netname: PASS (18.44 s)
 (2/2) type_specific.io-github-autotest-libvirt.virsh.net_start.normal_test.stop_start_firewalld.valid_netname: PASS (13.65 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 32.92 s

File: libvirt/tests/src/virsh_cmd/network/virsh_net_start.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import logging
+import time
 
 from virttest import virsh
 from virttest import libvirt_vm
 from virttest.libvirt_xml import network_xml, IPXML
 from virttest import libvirt_version
 from virttest.utils_test import libvirt
+from virttest.staging import service
+from virttest import utils_libvirtd
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_start.py:10]: We'd better have imports in alphabetical order, can you please update this part?

Comment (yalzhang) [reply libvirt/tests/src/virsh_cmd/network/virsh_net_start.py]: Updated, Thank you!

**Review by Yingshun — APPROVED** [review]
LGTM.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**